src/test/java/healthy/lifestyle/backend/activity/mental/controller/MentalWorkoutControllerTest.java

```diff
@@ -213,4 +215,147 @@ void createCustomMentalWorkout_shouldReturnErrorMessageWith400_whenMentalActivit
                 .andDo(print())
                 .andReturn();
     }
+
+    @Test
+    void getDefaultMentalWorkoutById_shouldReturnDefaultMentalWorkoutDtoWith200_whenValidRequest() throws Exception {
+        // Given
+        HttpRef defaultHttpRef1 = dbUtil.createDefaultHttpRef(1);
+        HttpRef defaultHttpRef2 = dbUtil.createDefaultHttpRef(2);
+        MentalType mentalType1 = dbUtil.createAffirmationType();
+        MentalType mentalType2 = dbUtil.createMeditationType();
+
+        MentalActivity defaultMental1 = dbUtil.createDefaultMentalActivity(1, List.of(defaultHttpRef1), mentalType1);
+        MentalActivity defaultMental2 = dbUtil.createDefaultMentalActivity(2, List.of(defaultHttpRef2), mentalType2);
+
+        MentalWorkout defaultMentalWorkout =
+                dbUtil.createDefaultMentalWorkout(1, List.of(defaultMental1, defaultMental2));
+
+        // When
+        MvcResult mvcResult = mockMvc.perform(get(URL.DEFAULT_MENTAL_WORKOUT_ID, defaultMentalWorkout.getId())
+                        .contentType(MediaType.APPLICATION_JSON))
+
+                // Then
+                .andExpect(status().isOk())
+                .andDo(print())
+                .andReturn();
+
+        String responseContent = mvcResult.getResponse().getContentAsString();
+        MentalWorkoutResponseDto responseDto =
+                objectMapper.readValue(responseContent, new TypeReference<MentalWorkoutResponseDto>() {});
+
+        assertThat(responseDto)
+                .usingRecursiveComparison()
+                .ignoringFields("mentalActivities")
+                .isEqualTo(defaultMentalWorkout);
```

Comment:
Consider checking whether there are two mental activities in the response 
`assertEquals(defaultMentalWorkout.getMentalActivites.size(), responseDto.getMentalActivites.size())`